src/slide-pane/styles/slide-pane.m.css

```diff
@@ -11,6 +11,13 @@
 	box-sizing: border-box;
 }
 
+.contentFixed {
+	overflow-y: auto;
+	position: fixed;
```

Comment:
I believe this should just be the `overflow-y` style, we wouldn't want to apply `position: fixed` or a set margin-top here

Reply:
Ok just using height 100% did the trick as well. But without it, it does not work.

Reply:
So I removed position:fixed; and margin-top with my lastest commit